.github/workflows/bump-dependencies.yml

```diff
@@ -137,7 +137,14 @@ jobs:
           fi
           if [[ $existing_pr ]]; then
             existing_title="$(gh pr view "${existing_pr}" --json title --jq .title)"
-            gh pr edit "${existing_pr}" --title "${pr_title}" --body "${body}"
+            # Use Github's API directly instead of using `gh edit`.
+            # The latter internally queries all fields of the PR which leads to
+            # a permission error as the workflow does not have access to
+            # organization projects:
+            gh api --silent -H "Accept: application/vnd.github+json" --method PATCH \
+              "/repos/${GITHUB_REPOSITORY}/pulls/${existing_pr}" \
+              -f title="${pr_title}" \
+              -f body="${body}"
```

Comment:
This appears as part of this PR as the last workflow run was from the PR branch for #2801. It will be fixed once that PR is merged and the workflow runs again.